pyiceberg/utils/config.py

```diff
@@ -84,12 +84,13 @@ def _load_yaml(directory: Optional[str]) -> Optional[RecursiveDict]:
                     return file_config_lowercase
             return None
 
-        # Give priority to the PYICEBERG_HOME directory
-        if pyiceberg_home_config := _load_yaml(os.environ.get(PYICEBERG_HOME)):
-            return pyiceberg_home_config
-        # Look into the home directory
-        if pyiceberg_home_config := _load_yaml(os.path.expanduser("~")):
-            return pyiceberg_home_config
+        # Directories to search for the configuration file
```

Comment:
nit: i like the comment on search order "PYICEBERG_HOME, home directory, then current directory"

Reply:
I added a new line to show this in https://github.com/apache/iceberg-python/pull/1464/commits/2bd69139da4585c88df73dc7f7a36903fc7daa7a, is this what you are looking for? I just added back this comment underneath, I hope this is what you're looking for.